enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/CausalClusteringSettings.java

```diff
@@ -124,6 +125,19 @@
             setting( "causal_clustering.initial_discovery_members", list( ",", ADVERTISED_SOCKET_ADDRESS ),
                     NO_DEFAULT );
 
+    @Description( "Type of in-flight cache." )
```

Comment:
Generally settings are public static final. This is more of a consistency thing but i find it nice.

Reply:
My mistake for avoiding the tried and true copy-n-paste!